src/fetchWrapper.js

```diff
@@ -1,7 +1,7 @@
-const fetch = require("node-fetch");
+const nodeFetch = require("node-fetch");
 
 // Custom API error to throw
-function ApiError(message, data, status) {
+function apiError(message, data, status) {
```

Comment:
If this is intended to act as a class (which it is), JS convention is to titlecase the classname.

```suggestion
function ApiError(message, data, status) {
```

OR, if you wanna get weird

```suggestion
function APIError(message, data, status) {
```

Reply:
🤦 I just noticed this inconsistency: we have both `ApiError` and `ImgixAPI`. Looks like initialisms should be all uppercased or all lowercased. [[1]](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms) [[2]](https://arcticicestudio.github.io/styleguide-javascript/rules/naming_conventions.html#acronyms-and-initialisms)